### PR TITLE
Created entity Todo + controller, added fixtures

### PR DESCRIPTION
**Pull request by StricTGod**

### CODE REVIEW

File: migrations/Version20220815191443.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE todo CHANGE is_completed is_completed TINYINT(1) DEFAULT 0 NOT NULL');
+    }
```
Comment (NerphMe) [inline migrations/Version20220815191443.php]: Move it to parent migration

File: src/Controller/TodoController.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ){
+        $this->todoRepository = $todoRepository;
+    }
+
+    /**
+     * @Route("/todo", name="todo", methods={"GET"})
+     */
+    public function index(): Response
+    {
+        $todos = $this->todoRepository->findAll();
```
Comment (NerphMe) [inline src/Controller/TodoController.php]: Take sorted todos by created at DESC

File: src/Repository/TodoRepository.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+//    public function findOneBySomeField($value): ?Todo
+//    {
+//        return $this->createQueryBuilder('t')
+//            ->andWhere('t.exampleField = :val')
+//            ->setParameter('val', $value)
+//            ->getQuery()
+//            ->getOneOrNullResult()
+//        ;
+//    }
+}
```
Comment (NerphMe) [inline src/Repository/TodoRepository.php]: Why do we need it?

File: templates/todo/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <tr>
+                <th>Name</th>
+                <th>Created_at</th>
+                <th>isCompleted</th>
+            </tr>
+            {% for todo in todos %}
+                <tr>
+                    <td>{{ todo.name }}</td>
+                    <td>{{ todo.createdAt|date('Y-m-d') }}</td>
+                    <td>{{ todo.isCompleted ? 'True' : 'False'}}</td>
```
Comment (NerphMe) [inline templates/todo/index.html.twig]: Inside interpolation u can use true or false without 'string'

File: migrations/Version20220815191443.php (hunk 1 above)
Comment (StricTGod) [reply migrations/Version20220815191443.php]: [OK] Already at the latest version ("DoctrineMigrations\Version20220815191443")                                        
How can i do it?

File: src/Repository/TodoRepository.php (hunk 3 above)
Comment (StricTGod) [reply src/Repository/TodoRepository.php]: Pubf add and remove were created by default, pubf getTodos, getTodoById i didnt know that i need them or not so i past them for future.
I should delete them?

File: src/Controller/TodoController.php (hunk 2 above)
Comment (NerphMe) [reply src/Controller/TodoController.php]: https://stackoverflow.com/questions/17116888/how-to-sort-findall-doctrines-method

File: migrations/Version20220815191443.php (hunk 1 above)
Comment (NerphMe) [reply migrations/Version20220815191443.php]: https://www.doctrine-project.org/projects/doctrine-migrations/en/3.5/index.html

**Review by NerphMe — APPROVED** [review]

**Comment by NerphMe** [comment]
Merge

**Pull request merged**